source/releasenotes/2025-01-15-php-53-55-eol.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "PHP 5.3 and 5.5 no longer available starting February 25, 2025"
+published_date: "2025-01-15"
+categories: [infrastructure, security, action-required]
+---
+As part of our continued effort to protect and secure customer sites, the Pantheon platform will no longer offer PHP versions 5.3 and 5.5 as of February 25, 2025.
+
+PHP 5.5 was declared end-of-life (EOL) by [the PHP Foundation](https://www.php.net/supported-versions.php) on July 21, 2016, more than eight years ago. EOL software does not receive security or feature updates, and could expose sites to attack if any vulnerabilities or exploits are discovered.
+
+We will soon provide long term support (LTS) security coverage for all available PHP versions on the platform.
```

Comment:
This statement is somewhat confusing given the paragraph above, specifically "all available PHP versions". I _think_ I am to read this explicitly as _all available PHP versions **on the platform**_, but that's not explicitly defined. I think it would be good to clarify this (or specify which versions will be LTS'd).

Reply:
Yeah some clarity here is a good idea but it might be best served by a follow up PR for related docs 

I thought about crosslinking to [this section](https://docs.pantheon.io/supported-drupal) because of the last paragraph: 
> Also included is continued support for Drupal 7-compatible runtime environments on the Pantheon Platform, including PHP, MySQL and other prerequisites.

Then I thought about crosslinking to the [PHP doc](https://docs.pantheon.io/guides/php) but it doesn't have 5.6 represented

Reply:
Updated this text.